### ISSUE TITLE
About SSL loss

### ISSUE THREAD

**Issue by splinter21** [ZhikangNiu/A-DMA#4]
### Checks

- [x] This template is only for research question, not usage problems, feature requests or bug reports.
- [x] I have thoroughly reviewed the project documentation and read the related paper(s).
- [x] I have searched for existing issues, including closed ones, no similar questions.
- [x] I am using English to submit this issue to facilitate community communication.

### Question details

1、which SSL do you use? official Hubert base? 
2、 hubert: 50hz (16k+hop320), f5 mel: 94hz (24k+hop256)  
how to align them? interploate?

**Comment by splinter21** [comment]
I found it .
x = F.interpolate(x.transpose(1, 2).contiguous(), size=ylens.max(), mode="linear")
